src/libs/ajax.js

```diff
@@ -1138,7 +1138,7 @@ const AzureStorage = signal => ({
     } else {
       const container = _.find(
         {
-          metadata: { resourceType: 'AZURE_STORAGE_CONTAINER' },
+          metadata: { resourceType: 'AZURE_STORAGE_CONTAINER', controlledResourceMetadata: { accessScope: 'SHARED_ACCESS' } },
```

Comment:
Another difference between the Leo container and the workspace one is that the Leo container has `managedBy: "APPLICATION"` vs. `managedBy: "USER"`. My thought was that checking the `accessScope` is sufficient, but we could check both if we wish. @aherbst-broad do you have an opinion?

Reply:
I think accessScope is ok for now.